_partials/_migrate_live_migration_step2.md

```diff
@@ -0,0 +1,54 @@
+<Highlight type="note">
+For sake of convenience, set `$SOURCE` and `$TARGET` connection strings
+in your migration instance.
+
+```sh
+export SOURCE="postgres://user:password@host:port/database"
+export TARGET="postgres://user:password@host:port/database"
+```
+</Highlight>
```

Comment:
This duplicated the content which is already in a partial, `./_partials/_migrate_set_up_source_and_target.md` please use that instead.